### ISSUE TITLE
Skip navigation button not focussing on content when navigating with arrows.

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-core#433]
### Subject of the issue/enhancement/features
When you have foccused on the content of a page, if you navigate back to the skip navigation button using arrows and select the button, it won't jump back to the content. 

### Your environment
* FW v5.31.21
* Chrome

### Steps to reproduce
Create a course, navigate to a page, select the skip navigation, navigate back to the skip navigation button using the arrows and select again. 

### Expected behaviour
Screen reader/focus should be jumped back to the content.

### Actual behaviour
The button does nothing and the screen reader will remain at the top of the navigation.

**Comment by swashbuck** [comment]
@joe-allen-89 Are you using a specific screenreader (ex. JAWS or NVDA)? I am not having this problem on Mac VoiceOver.

**Comment by joe-allen-89** [comment]
> @joe-allen-89 Are you using a specific screenreader (ex. JAWS or NVDA)? I am not having this problem on Mac VoiceOver.

Apologies thought I'd added it to the ticket, using Chrome with a combination of NVDA and Jaws 2023, initially flagged by QA.
